GPU/GLES/DrawEngineGLES.cpp

```diff
@@ -1119,54 +1119,96 @@ bool DrawEngineGLES::IsCodePtrVertexDecoder(const u8 *ptr) const {
 }
 
 void DrawEngineGLES::TessellationDataTransferGLES::SendDataToShader(const float *pos, const float *tex, const float *col, int size, bool hasColor, bool hasTexCoords) {
-#ifdef USING_GLES2
-#define GL_TEXTURE_1D GL_TEXTURE_2D
-#define glTexImage1D(t, l, ifmt, w, br, fmt, type, p) glTexImage2D(t, l, ifmt, w, 1, br, fmt, type, p)
-#define glTexSubImage1D(t, l, xo, w, fmt, type, p) glTexSubImage2D(t, l, xo, 0, w, 1, fmt, type, p)
-#endif
-	// Position
-	glActiveTexture(GL_TEXTURE3);
-	glBindTexture(GL_TEXTURE_1D, data_tex[0]);
-	glTexParameteri(GL_TEXTURE_1D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
-	glTexParameteri(GL_TEXTURE_1D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
-	glTexParameteri(GL_TEXTURE_1D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
-	glTexParameteri(GL_TEXTURE_1D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
-	if (prevSize < size) {
-		glTexImage1D(GL_TEXTURE_1D, 0, GL_RGB32F, size, 0, GL_RGB, GL_FLOAT, (GLfloat*)pos);
-		prevSize = size;
-	} else {
-		glTexSubImage1D(GL_TEXTURE_1D, 0, 0, size, GL_RGB, GL_FLOAT, (GLfloat*)pos);
-	}
-
-	// Texcoords
-	if (hasTexCoords) {
-		glActiveTexture(GL_TEXTURE4);
-		glBindTexture(GL_TEXTURE_1D, data_tex[1]);
+	if (isAllowTexture1D) {
```

Comment:
Build fails, because you can't really replace that #ifdef with a branch that checks for isAllowTexture1D. Doesn't matter if it's taken or not, both ways have to be compiled and why it still needs those definitions/failing without them.